ext/url/00_url.js

```diff
@@ -436,20 +435,21 @@
         prefix,
         context: "Argument 1",
       });
-      this[_url] = opUrlParse(value);
+      this[_rid] = opUrlParse(value);
+      this[_url] = new Array(11);
```

Comment:
This will create a holey array. I'm not sure how much worse that is in terms of access-by-index performance, but I would consider doing either a manual creation:
```js
this[_url] = [undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined];
```
or possibly having such an array in the module scope and making a copy of it here.

Reply:
good point, just went with manual declaration to avoid primordials